src/util.coffee

```diff
@@ -32,3 +32,25 @@
                                       node?.tag is 'tag:yaml.org,2002:map'
 
 @NON_PRINTABLE = /[^\x09\x0A\x0D\x20-\x7E\x85\xA0-\uD7FF\uE000-\uFFFD]/
+
+@stringifyJSONCircularDepth = (json, depth = 1000) ->
```

Comment:
I don't agree ! With a schema like this one you will abort the dereferencing parsing?
```json
{
    "$schema": "http://json-schema.org/schema",
    "definitions": {
        "node": {
            "type": "object",
            "properties": {
                "id": {
                    "type": "string",
                    "format": "id"
                },
                "label": {
                    "type": "string"
                },
                "children": {
                    "type": "array",
                    "items": {
                        "$ref": "#/definitions/node"
                    }
                }
            },
            "required": ["id", "label", "children"]
        }
    },
    "items": {
        "$ref": "#/definitions/node"
    },
    "type": "array"
}
```
If I want generate mocks from this schema how should I do it?